packages/dwm/dwm.c.patch

```diff
@@ -0,0 +1,11 @@
+--- dwm-6.3/dwm.c	2022-01-07 06:42:18.000000000 -0500
++++ dwm-mod/dwm.c	2022-02-28 19:36:32.859355732 -0500
+@@ -838,7 +838,7 @@
+ {
+ 	Client *c = NULL, *i;
+ 
+-	if (!selmon->sel || (selmon->sel->isfullscreen && lockfullscreen))
++	if (!selmon->sel || (selmon->sel->isfullscreen && setfullscreen))
```

Comment:
Any explanation about this change?

Reply:
In the local build it works without problems, but from GitHub actions I launched that error, I had to make that patch to fix it

Reply:
Isn't it sufficient to add the following line

```c
static const int lockfullscreen = 1; /* 1 will force focus on the fullscreen window */
```

just below line 37 of `packages/dwm/config.h`:

https://github.com/termux/x11-packages/blob/71b8524fde1cf9cc4d44a853554a3fff23692d4f/packages/dwm/config.h#L37

Reply:
And I insist that `packages/dwm/dwm.c.patch` be removed.

Reply:
Oh well, tomorrow I do, Now I'm very sleepy